docs/reference/terminal49/terminal49.v1.json

```diff
@@ -5664,6 +5664,9 @@
                   "container.transport.vessel_discharged",
                   "container.transport.full_out",
                   "container.transport.empty_in",
+                  "container.transport.rail_arrived",
+                  "container.transport.rail_loaded",
+                  "container.transport.rail_unloaded",
```

Comment:
Lets make the order here and elsewhere:
- loaded
- departed
- arrived
- unloaded